armada-ths/ais
# Banquet flow

Created a banquet dummy flow.
Changed on_deletion behavior for BanquetTables to stop them from deleting banquetattendants.

To test:
* host server
* goto localhost:8000/fairs/2017/banquet/
* make sure you have at least 1 banquet attendant
* press the "sit attendants" button on the top
* if you don't see the button, log in on an account that has banquet.banquetattendant.can_sit_attendants permission or give the account one.
* make sure the attendants got sitted at tables. (Will use existing tables with correct sizes, or create new ones of size 8 as needed)
